File: app/src/main/java/edu/usc/nlcaceres/infectionprevention/ActivitySettings.java
package edu.usc.nlcaceres.infectionprevention;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import edu.usc.nlcaceres.infectionprevention.databinding.ActivitySettingsBinding;
import edu.usc.nlcaceres.infectionprevention.util.ViewHelpersKt;

public class ActivitySettings extends AppCompatActivity {
    private ActivitySettingsBinding viewBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        viewBinding = ActivitySettingsBinding.inflate(getLayoutInflater());
        setContentView(viewBinding.getRoot());

        // Calling Kotlin code from Java means default param values are REQUIRED, no omitting like in kotlin!
        ViewHelpersKt.SetupToolbar(this, viewBinding.toolbarLayout.homeToolbar,
            R.drawable.ic_back_arrow, "Settings");

        getSupportFragmentManager().beginTransaction().replace(R.id.settings_fragment, FragmentSettings.newInstance()).commit();
    }

}

File: app/src/main/java/edu/usc/nlcaceres/infectionprevention/ActivitySettings.kt
package edu.usc.nlcaceres.infectionprevention

import edu.usc.nlcaceres.infectionprevention.util.SetupToolbar
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import edu.usc.nlcaceres.infectionprevention.databinding.ActivitySettingsBinding

class ActivitySettings : AppCompatActivity() {
  private lateinit var viewBinding: ActivitySettingsBinding

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    viewBinding = ActivitySettingsBinding.inflate(layoutInflater)
    setContentView(viewBinding.root)

    SetupToolbar(this, viewBinding.toolbarLayout.homeToolbar, R.drawable.ic_back_arrow, "Settings")

    supportFragmentManager.beginTransaction()
      .replace(R.id.settings_fragment, FragmentSettings.newInstance()).commit()
  }
}